README.md

```diff
@@ -9,3 +9,4 @@ turecross321/turecross321 is a ✨ special ✨ repository because its `README.md
 You can click the Preview link to take a look at your changes.
 --->
  so yeah thats basically it
+
```

Comment:
Definitely not the way I would do it. I would do this much cleaner. But this might work anyway. This will have to work. But I would definitely do this cleaner. I'll let you off the hook this time. I would do this cleaner, but I feel nice today, so I will let it slide. Don't forget that I would do this substantially cleaner.